polling_stations/apps/data_collection/management/commands/import_bradford.py

```diff
@@ -6,32 +6,75 @@ class Command(BaseXpressDemocracyClubCsvImporter):
     # you manually created this file to include one edit:
     # Polling district 12B
     # will be at St Martin’s Church, Haworth Road, Bradford, BD9 6LL
```

Comment:
I would delete these comments as I'm pretty sure they applied to the Euros and no longer apply.

Reply:
Yep, done.